src/test/scala/edu/ie3/simona/io/file/ResultFileHierarchySpec.scala

```diff
@@ -106,37 +83,22 @@ class ResultFileHierarchySpec
       ) shouldBe true
 
       // check for existence of other folders
-      assert(
-        Files.exists(new File(validOutputFileHierarchy.baseOutputDir).toPath)
-      )
-      assert(Files.exists(new File(validOutputFileHierarchy.tmpDir).toPath))
-      assert(
-        Files.exists(new File(validOutputFileHierarchy.configOutputDir).toPath)
-      )
-      assert(
-        Files.exists(new File(validOutputFileHierarchy.rawOutputDataDir).toPath)
-      )
-      assert(
-        Files.exists(new File(validOutputFileHierarchy.graphOutputDir).toPath)
-      )
-      assert(
-        Files.exists(new File(validOutputFileHierarchy.kpiOutputDir).toPath)
-      )
+      assert(Files.exists(validOutputFileHierarchy.configOutputDir))
+      assert(Files.exists(validOutputFileHierarchy.logOutputDir))
+      assert(Files.exists(validOutputFileHierarchy.tmpDir))
 
       // check if tmp directory can be deleted by output file hierarchy
       ResultFileHierarchy.deleteTmpDir(validOutputFileHierarchy)
-      assert(!Files.exists(new File(validOutputFileHierarchy.tmpDir).toPath))
+      assert(!Files.exists(validOutputFileHierarchy.tmpDir))
 
     }
 
     "should add a timestamp automatically on instantiation" in {} // todo
 
   }
 
-  private def relativizePath(fullPath: String): Path = {
-    new File(new File("").getAbsolutePath).toPath
-      .relativize(new File(fullPath).toPath)
-  }
+  private def relativizePath(fullPath: Path): Path =
+    new File(new File("").getAbsolutePath).toPath.relativize(fullPath)
```

Comment:
I think this method can be removed, since it is not used anywhere.

Reply:
Good catch!